mix.exs

```diff
@@ -40,12 +40,12 @@ defmodule Readability.Mixfile do
   # Type "mix help deps" for more examples and options
   defp deps do
     [
-      {:floki, "~> 0.20"},
-      {:httpoison, "~> 0.13.0"},
-      {:ex_doc, "~> 0.14", only: :dev},
-      {:credo, "~> 0.9", only: [:dev, :test]},
+      {:credo, "~> 1.1.5", only: [:dev, :test]},
       {:dialyxir, "~> 0.3", only: [:dev]},
-      {:mock, "~> 0.2.0", only: :test}
+      {:ex_doc, "~> 0.21.2", only: :dev},
+      {:floki, "~> 0.23.0"},
+      {:httpoison, "~> 1.6.1"},
```

Comment:
this is too restrictive, there is no need to restrict httpoison to 1.6.1.
I would do: `{:httpoison, "~> 1.5"}`

Reply:
i agree, should i create a new PR?

Reply:
I recently became maintainer of this lib. Closing this PR as old, since we'll use one of the newer PRs to update dependencies. Thanks for the contribution, though!